torch/_prims/__init__.py

```diff
@@ -1633,6 +1633,23 @@ def _convert_element_type_nvfuser(fd: Any, a: Tensor, dtype: torch.dtype) -> Ten
     doc=_convert_element_type_doc,
 )
 
+# NOTE: _to_dtype
+# This private op casts the input to the desired type while preserving its stride
+# permutation, unlike .to(dtype) which will create a tensor with contiguous strides
+# TODO: review how best to model this behavior
+def _to_dtype(a: TensorLikeType, dtype: torch.dtype) -> TensorLikeType:
+    # FIXME: improve this so it doesn't error on Meta testing
+    try:
+        requires_grad = a.requires_grad
+    except Exception as e:
+        requires_grad = False
```

Comment:
@eellison if we fully replace TensorMeta with FakeTensor I think it will fix this